src/AppInstallerCLICore/Workflows/ShowFlow.cpp

```diff
@@ -12,8 +12,28 @@ using namespace AppInstaller::Utility;
 using namespace AppInstaller::Utility::literals;
 
 namespace {
-    void ShowMultiLineField(Execution::OutputStream& outputStream, AppInstaller::StringResource::StringId label, std::string& value)
+
+    template <typename CharT, typename String>
+    void ShowSingleLineField(Execution::OutputStream& outputStream, CharT label, String value, bool indent = false)

```

Comment:
I didn't see anything but `StringID` values for label (and I wouldn't expect anything else).  I don't think it needs to be a templated type.  Applies to `ShowMultiLineField` as well.

And it's best to use `const String& value` to show the intent (we do not want to modify or own this parameter).